### ISSUE TITLE
why apk file is downloaded as zip file?

### ISSUE THREAD

**Issue by jianmaikj** [mifi/ezshare#88]
I'm sharing an APK files to an Android device, but it is downloaded as a zip

**Comment by mifi** [comment]
what do you mean by that? the file name extension gets renamed frok `.apk` to `.zip` but the file is still an APK?

**Comment by jianmaikj** [comment]
> what do you mean by that? the file name extension gets renamed frok `.apk` to `.zip` but the file is still an APK?

yes, it was renamed from `.apk` to `.zip` automatically when I downloaded it

**Comment by mifi** [comment]
could be mime-type/content-type related
